packages/app/src/containers/ThemeTextStyleInspector.tsx

```diff
@@ -0,0 +1,236 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import { Divider } from 'noya-designsystem';
+import { Selectors } from 'noya-state';
+import { memo, useCallback, useMemo } from 'react';
+import {
+  useApplicationState,
+  useSelector,
+} from '../contexts/ApplicationStateContext';
+import useShallowArray from '../hooks/useShallowArray';
+import FillInspector from './FillInspector';
+import withSeparatorElements from '../utils/withSeparatorElements';
+import NameInspector from '../components/inspector/NameInspector';
+import OpacityInspector from './OpacityInspector';
+import BorderInspector from './BorderInspector';
+import ShadowInspector from './ShadowInspector';
+import { delimitedPath } from 'noya-utils';
+import TextOptionsRow, {
+  SimpleTextDecoration,
+} from '../components/inspector/TextOptionsRow';
+import TextAlignmentRow from '../components/inspector/TextLayoutRow';
+import TextStyleRow from '../components/inspector/TextStyleRow';
+import getMultiValue from '../utils/getMultiValue';
+
+const TextStyleInspector = memo(function TextStyleInspector() {
+  const [, dispatch] = useApplicationState();
+
+  const seletedText = useShallowArray(useSelector(Selectors.getSelectedStyles));
+
+  const getTextStyleAttributes = useCallback((styles: Sketch.Style[]) => {
```

Comment:
Is this the selector you mentioned in your comment? 

Maybe as a starting point, we can write a helper function in `noya-state/src/textStyleSelectors.ts` that extracts just the properties shared between the two inspectors? 

The `selectors.ts` file is getting large, so probably I will move it into a directory and split things into separate files, but I want to wait till I see Leanne's PR in case it causes conflicts

Reply:
I created the file 😅  I hope it's not too far of to what you meant. With this selector I was able to combine both Inspector in one.